contrib/jabber/packages.el

```diff
@@ -0,0 +1,42 @@
+;;; packages.el --- jabber Layer packages File for Spacemacs
+;;
+;; Copyright (c) 2012-2014 Sylvain Benner
+;; Copyright (c) 2014-2015 Sylvain Benner & Contributors
+;;
+;; Author: Tosh Lyons <tosh.lyons@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(defvar jabber-packages
+  '(
+    jabber
+    ))
+
+;; List of packages to exclude.
+(defvar jabber-excluded-packages '())
+
+(defun jabber/init-jabber ()
+  "Initialize jabber"
+  (use-package jabber
+    :defer t
+    :init
+    (evil-leader/set-key
+      "aj" 'jabber-connect-all)
+    :config
+    (progn
+      (evil-leader/set-key-for-mode 'jabber-roster-mode
```

Comment:
You should put these keybindings under the `SPC m` prefix that is designated for mode-specific bindings.